app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

```diff
@@ -0,0 +1,427 @@
+---
+title: Self-service registration for accrediting bodies and their training providers
```

Comment:
```suggestion
title: Let users at accredited bodies set up an account for themselves and users at their partner training providers
```

Reply:
We refer to "higher education institutions" rather than "accredited bodies" since the latter could also include SCITTS, which aren't part of this work. We've also used "school direct partners" in relation to training providers to be more explicit.

I will combine your suggestion with the preferred nomenclature.